src/Panel/Router.php

```diff
@@ -159,15 +159,15 @@ public static function routes(array $areas): array
 		];
 
 		// register all routes from areas
-		foreach ($areas as $id => $area) {
+		foreach ($areas as $area) {
 			$routes = [
 				...$routes,
-				...static::routesForViews($id, $area),
-				...static::routesForSearches($id, $area),
-				...static::routesForDialogs($id, $area),
-				...static::routesForDrawers($id, $area),
-				...static::routesForDropdowns($id, $area),
-				...static::routesForRequests($id, $area),
+				...static::routesForViews($area),
+				...static::routesForSearches($area),
+				...static::routesForDialogs($area),
+				...static::routesForDrawers($area),
+				...static::routesForDropdowns($area),
+				...static::routesForRequests($area),
 			];
 		}
```

Comment:
`Nitpick:` Not necessarily in this PR, but looking at this it just screams to me for 

```php
...$area->routes()
```